src/sentry/static/sentry/app/views/settings/organizationAuditLog/index.jsx

```diff
@@ -46,11 +46,11 @@ const EVENT_TYPES = [
   'rule.create',
   'rule.edit',
   'rule.remove',
-  'serivcehook.create',
-  'serivcehook.edit',
-  'serivcehook.remove',
-  'serivcehook.enable',
-  'serivcehook.disable',
+  'servicehook.create',
+  'servicehook.edit',
+  'servicehook.remove',
+  'servicehook.enable',
+  'servicehook.disable',
```

Comment:
Is the old name stored in the database anywhere?

Reply:
In a slack discussion with Colleen she confirmed that we store integers in the database and not these strings :+1: